### PR TITLE
[SDRAN-1384]  Initial sidecar deployment for model plugins

### PR DESCRIPTION
**Pull request by teone**

### CODE REVIEW

File: onos-config/templates/_helpers.tpl [hunk 1]
```diff
@@ -117,3 +117,16 @@ openpolicyagent image name
 {{- .Values.openpolicyagent.image.tag -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+  modelPlugins image names,
+  takes the image name and the .Values to add the registry if any.
+*/}}
+{{- define "modelplugin.imagename" -}}
+{{- if .Values.global.image.registry -}}
```
Comment (teone) [inline onos-config/templates/_helpers.tpl:126]: Is it ok to use the same registry as the `onos-config` image or do we want to specify a different one? I can't really imagine a usecase in which images come from a different registry, but who knows...

Comment (tomikazi) [reply onos-config/templates/_helpers.tpl:126]: Hmmm... I think for now, let's use the same registry. If the usecase emerges for supporting different ones, we'll cross that bridge then.

**Review by tomikazi — APPROVED** [review]

File: onos-config/values.yaml [hunk 2]
```diff
@@ -35,6 +35,14 @@ debug: false
 
 devices: []
 
+# model plugins list
+# deployed as a sidecar to onos-config, eg:
+
+# modelPlugins:
```
Comment (adibrastegarnia) [inline onos-config/values.yaml:41]: Should we have some default model plugins here? 

@teone  @tomikazi

**Comment by adibrastegarnia** [comment]
It should be rebased to get new version of onos-config chart.

File: onos-config/values.yaml (hunk 2 above)
Comment (tomikazi) [reply onos-config/values.yaml:41]: I use these by default:
modelPlugins:
  - name: devicesim1
    image: onosproject/devicesim:1.0.0
    port: 5152
  - name: testdevice1
    image: onosproject/testdevice:1.0.0
    port: 5153
  - name: testdevice2
    image: onosproject/testdevice:2.0.0
    port: 5154

Comment (adibrastegarnia) [reply onos-config/values.yaml:41]: yeah we should put them there

Comment (teone) [reply onos-config/values.yaml]: Done

File: onos-config/templates/_helpers.tpl (hunk 1 above)
Comment (teone) [reply onos-config/templates/_helpers.tpl:126]: Sounds good

**Comment by adibrastegarnia** [comment]
retest this please

**Review by tomikazi — APPROVED** [review]

**Pull request merged**